app/javascript/react/components/Modals/TimelapseModal/TimelapseComponent.tsx

```diff
@@ -193,10 +159,7 @@ const TimelapseComponent: React.FC<
                 timestamps={filteredTimestamps}
               />
             </S.TimeAxisContainer>
-            <TimeRangeButtons
-              timeRange={timeRange}
-              onSelectTimeRange={setTimeRange}
-            />
+            <TimeRangeButtons timeRange={timeRange} />
```

Comment:
Do we need to retain the timeRange prop since with the changes you made the timeRange is managed in redux?

Reply:
good point 👍